test/services/GuidanceServiceSpec.scala

```diff
@@ -56,9 +57,10 @@ class GuidanceServiceSpec extends BaseSpec {
     "not retrieve a page from the process" in new Test {
       val processId = "ext90001"
       val url = "scooby"
-      MockGuidanceConnector
-        .getProcess(processId)
-        .returns(Future.successful(process))
+
```

Comment:
We could have a test where the session repository fails to return a process.

Reply:
Will do